spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/StreamListenerResultAdapter.java

```diff
@@ -41,6 +43,6 @@
 	 * @param streamListenerResult the result of invoking the method.
 	 * @param bindingTarget the binding target.
 	 */
-	void adapt(R streamListenerResult, B bindingTarget);
+	Closeable adapt(R streamListenerResult, B bindingTarget);
```

Comment:
Due to the fact that we have other breaking changes (e.g. dependency requirements or so) I think that it is safe to to do this, especially as this never was a publicly used API.